### DISCUSSION TITLE
第6节课笔记(1班)

### DISCUSSION THREAD

**Discussion by AXYZdong** [InternLM/Tutorial#394] [笔记]
请在下方提交第 **6** 节课笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135719669

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/21/ai-internlm-opencompass/

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap5.md

**Comment by strong-leaf** [comment]
https://zhuanlan.zhihu.com/p/678896060
笔记在前

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135727570

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/678998336

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/679037817

**Comment by Gu-Xiao** [comment]
https://www.bilibili.com/read/cv29872599/?jump_opus=1

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135719438

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135742273

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135759716

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/881f18e760cc47e78230001b745f60ee

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135763811

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb?ccm_open_type=lark_docs_home_Recent&from=tab_home#part-YB7Idzv9zoBoxbxRwrLcV4mInIb

**Comment by gongxiucheng** [comment]
https://ckfajwu6zv.feishu.cn/docx/GuhFdjcnCoMJaZxCePbcA3zdnLg?from=from_copylink
